data/hai/unfettered 0 prologue.txt

```diff
@@ -506,57 +509,1298 @@ mission "Unfettered Jump Drive Trading"
+			`	When landing on an Unfettered planet, even after having brought them a jump drive, there is always some lingering tension. Now you feel the tension much more than usual, probably because of the medical emergency. Unfettered rush around, and various vehicles are frequently coming and going, bringing both supplies and the infected to the hospital. The hospital itself looks different than the rest of the Unfettered structures: it seems more organized, proper. It is probably the best maintained structure here alongside the shipyards - or at least it has that appearance. One part of it looks to have been grafted onto the rest though, and seems to be a recent addition. Not even their critical buildings appear to be totally safe from the Unfettered's propensity for haphazard building extensions.`
+			`	Many pilots seem to have taken up the call, but it appears the organizers here are still interested in all the volunteers they can get. Now that you're here, you're not sure you'll be able to leave without at least being accosted for your assistance with this.`
+			choice
+				`	(Signal them that I'm going to get the supplies.)`
+					goto agree
+				`	(I don't have time for this.)`
+			`	You manage to leave without anyone stopping you in all the rush, though you're sure someone was trying to get your attention. If they remember you, the Unfettered will likely not be too happy. You probably also shouldn't stay landed here too long in case they decide to follow you.`
+				decline
+			label agree
+			`	You are quickly assigned the collection of supplies from <stopovers> and urged to return them as quickly as possible.`
+				accept
+	on stopover
+		dialog `You're surprised to see the Hai are more than happy to give you the medical supplies needed. They load up the cargo and, once they are done, one of them addresses you. "Tell our brothers and sisters to stay strong. Hopefully it is not too late," he says with determination.`
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on visit
+		dialog phrase "generic missing stopover or cargo"
+	on complete
+		log `Brought medial supplies to Darkcloak to prevent an epidemic.`
+		log "People" "Sookootow" `An Unfettered Chief worried for his people, inside Hai or Unfettered territory. He seems to believe that all Hai are equal, and as a member of the council he has significant influence.`
+		payment 1500000
+		conversation
+			`The staff seem greatly relieved when you return promptly with the medical supplies. As you're returning to your ship, one of them catches up to you. "Hello, captain <last>. I am Merki, manager of this hospital. I wanted to express my deepest thanks to you and give you the <payment> personally. When you have the time, make sure to check the job board for more jobs like this, it would mean a great deal to me."`
```

Comment:
```suggestion
			`The staff seem greatly relieved when you return promptly with the medical supplies. As you're returning to your ship, one of them catches up to you. "Hello, captain <last>. I am Merki, manager of this hospital. I wanted to express my deepest thanks to you and give you the <payment> personally. I find it quite admirable that an outsider would risk getting caught up in all of this. It would mean a great deal to me, if your bravery holds fast, so make sure to check the job board for more jobs like this."`
```